test_integration/scripts/base_token/deploy_subgraph.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+originChain="dev-origin"
+auxiliaryChain="dev-auxiliary"
+auxChainIdentifier="1000"
+
+echo deploying baseToken sub-graph for $originChain chain
+./node_modules/.bin/mosaic subgraph $originChain $auxChainIdentifier origin http://localhost:9535 http://localhost:6516
```

Comment:
Read the hardcoded graph node host/port from constants/variables?

Reply:
I think it's OK for tests to hardcode URLs.